server/users/user.service.js

```diff
@@ -30,7 +30,11 @@ async function getAll() {
 }
 
 async function getById(id) {
-  return await User.findById(id);
+  const user = await User.findById(id)
+  if(user)
+  return {...user.toJSON()};
```

Comment:
instead, you can also make a toObject method and it'll convert it automatically as well